frontend/web/styles/3rdParty/_hljs.scss

```
@@ -296,6 +277,23 @@ code.txt {
 }
 
 code[contenteditable='false'].hljs {
-  cursor: default !important;
+  cursor: default;
   @include customScrollDark();
```

Stylelint (17.14.0)

[error] Unexpected parentheses in argumentless mixin "customScrollDark" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)